### PR TITLE
Issue #1638 by kasperg: Update Drupal Core to 7.42

### PR DESCRIPTION
**Pull request by kasperg**

### CODE REVIEW

**Comment by guddo** [comment]
I think you should update the README.md accordingly (also the patches section).

File: drupal.make [hunk 1]
```diff
@@ -5,7 +5,7 @@ api = 2
 ; As d.o is having issues with the update XML file, we are using this form for downloading core.
 ; See this: https://drupal.org/node/2126123
 projects[drupal][type] = core
-projects[drupal][version] = 7.41
+projects[drupal][version] = 7.42
 projects[drupal][download][type] = get
 projects[drupal][download][url] = http://ftp.drupal.org/files/projects/drupal-7.41.tar.gz
```
Comment (cableman) [inline drupal.make]: You need to change the version for the tar file as well :-)

**Comment by kasperg** [comment]
@guddo Agreed. Done. Having the version mentioned explicitly in the `README` is not sustainable. I will try to to update it so we have fewer references to it.

**Comment by kasperg** [comment]
Closed in favor of #162. 7.43 is actually the latest version of Drupal Core containing security fixes.

**Pull request closed without merging**